template/src/utilities/networking.js

```diff
@@ -0,0 +1,102 @@
+import instance from "./axiosInstance";
+
+export function setHeader (property, data) {
+  instance.defaults.headers.common[property] = data;
+};
+
+export function setBaseURL (baseURL) {
+  instance.defaults.baseURL = baseURL;
+};
+
+export function setToken (token, type = null) {
+    switch (type) {
+      case "Paymentez": {
```

Comment:
Could we create `enum` for this value. Don't hard code the string

Reply:
done